### PR TITLE
Include Travis CI files and First Test

### PR DESCRIPTION
**Pull request by bh2smith**

### CODE REVIEW

**Comment by fleupold** [comment]
Is the failed travis build expected?

**Comment by bh2smith** [comment]
@fleupold It worked once, but now it is failing on "npm test" because, presumably, we don't have any tests. I am in the process of tryign to figure out whats up. Once it is passing I will submit the merge request to you!

**Comment by josojo** [comment]
let's get some tests added and then we can merge it. looks good

**Comment by bh2smith** [comment]
Hey all, Travis tests are running and passing with some borrowed PriorityQueue Tests!

File: migrations/3_plasma_migration.js [hunk 1]
```diff
@@ -1,8 +1,9 @@
 
 const EtherToken = artifacts.require("EtherToken")
 const Plasma = artifacts.require("Plasma.sol")
-
+const PriorityQueue = artifacts.require("PriorityQueue.sol");
 
 module.exports = function(deployer, networks, accounts) {
+    deployer.deploy(PriorityQueue)
```
Comment (josojo) [inline migrations/3_plasma_migration.js]: This deployment is only needed for testing, right? All other priority queues are deployed from the plasma contract.

Then, please, remove it from here and add this deployment to the test script. You can simply put in a before statement

Comment (bh2smith) [reply migrations/3_plasma_migration.js]: I am not sure I understand this. What I have in the test script appears to attempt to deploy this, but was unsuccessful


Observe, the first 20 lines of the test code:

```
let PriorityQueue = artifacts.require("PriorityQueue");

let {
    catchError,
    toHex,
    fastForward,
    proofForDepositBlock,
    zeroHashes
} = require('./utilities.js');

contract('PriorityQueue', async (accounts) => {
    let instance;
    before (async () => {
        instance = await PriorityQueue.deployed();
    });

```

Comment (bh2smith) [reply migrations/3_plasma_migration.js]: This is what happens when I remove that deployment line of code

```

  Contract: PriorityQueue
    1) "before all" hook


  0 passing (61ms)
  1 failing

  1) Contract: PriorityQueue
       "before all" hook:
     Error: PriorityQueue has not been deployed to detected network (network/artifact mismatch)
      at /usr/local/lib/node_modules/truffle/build/webpack:/packages/truffle-contract/contract.js:454:1
      at process._tickCallback (internal/process/next_tick.js:68:7)
```

Comment (bh2smith) [reply migrations/3_plasma_migration.js]: This has been adjusted in most recent commit.

**Review by josojo — APPROVED** [review]

**Pull request merged**